README.md

```diff
@@ -264,6 +272,10 @@ Renders the widget to the DOM, and passes control back to your app through succe
 - `success` *(optional)* - Function that is called when the user has completed an authentication flow. If an [OpenID Connect redirect flow](#openid-connect) is used, this function can be omitted.
 - `error` *(optional)* - Function that is called when the widget has been initialized with invalid config options, or has entered a state it cannot recover from. If omitted, a default function is used to output errors to the console.
 
+**OIDC Configuration:**
```

Comment:
Will devs get same (similar) possible `status` regardless OIDC or Non-OIDC config?

Reply:
I believe the `status` outcomes are the same, regardless of either config. I've left comments in the Non-OIDC config to the effect: https://github.com/okta/okta-signin-widget/pull/1473/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R364